docs/frameworks/symfony.md

```diff
@@ -154,46 +103,51 @@ To learn more about all this, read the [environment variables documentation](/do
 
 ## Trust API Gateway
 
-When hosting your site on Lambda, API Gateway will act as a proxy between the client and your function. 
+When hosting your site on Lambda, API Gateway will act as a proxy between the client and your function.
 
-By default, Symfony doesn't trust proxies for security reasons, but it's safe to do it when using API Gateway and
-Lambda. 
+By default, Symfony doesn't trust proxies for security reasons, but it's safe to do it when using API Gateway and Lambda.
 
 This is needed because otherwise, Symfony will not be able to generate URLs properly.
 
 You should add the following lines to `config/packages/framework.yaml`
 
 ```yaml
+# config/packages/framework.yaml
+
 framework:
-    # trust the remote address because API Gateway has no fixed IP or CIDR range that we can target
-    trusted_proxies: '127.0.0.1,REMOTE_ADDR'
-    # trust "X-Forwarded-*" headers coming from API Gateway
-    trusted_headers: ['x-forwarded-for', 'x-forwarded-proto', 'x-forwarded-port']
+  # trust the remote address because API Gateway has no fixed IP or CIDR range that we can target
+  trusted_proxies: '127.0.0.1,REMOTE_ADDR'
```

Comment:
Isn't `$_SERVER['REMOTE_ADDR']` always 127.0.0.1 on Lambda?

Reply:
I think it is https://github.com/brefphp/bref/blob/master/src/Event/Http/FpmHandler.php#L184

So we could change it to only keep the IP

```suggestion
  trusted_proxies: '127.0.0.1'
```

Reply:
:+1: